catalog/core/catalog-core-api/src/main/java/ddf/catalog/util/Describable.java

```diff
@@ -1,10 +1,10 @@
 /**
```

Comment:
No real edits to this file... just a reformat for the <p/>'s ... remove from PR?